app/Models/Beatmapset.php

```diff
@@ -485,6 +486,7 @@ public static function searchDB(array $params = [])
 
     public static function search(array $params = [])
     {
+        $startTime = microtime(true);
```

Comment:
perhaps just update the beatmapset search to use `Search` as well? Otherwise this search will be recorded twice when run on global search page.

Reply:
Yeah that looks non-trivial relative to the quick instrumentation I'm adding, feel free to refactor it later...